pyccl/boltzmann.py

```diff
@@ -174,6 +181,169 @@ def get_camb_pk_lin(cosmo):
     return pk_lin
 
 
+def get_isitgr_pk_lin(cosmo):
+    """Run ISiTGR-CAMB and return the linear power spectrum.
+
+    Args:
+        cosmo (:class:`~pyccl.core.Cosmology`): Cosmological
+            parameters. The cosmological parameters with
+            which to run ISiTGR-CAMB.
+
+    Returns:
+        :class:`~pyccl.pk2d.Pk2D`: Power spectrum \
+            object. The linear power spectrum.
+    """
+
+    assert HAVE_ISITGR, (
+        "You must have the `isitgr` python package "
+        "installed to run CCL with ISiTGR-CAMB!")
+
+    # z sampling from CCL parameters
+    na = lib.get_pk_spline_na(cosmo.cosmo)
+    status = 0
+    a_arr, status = lib.get_pk_spline_a(cosmo.cosmo, na, status)
+    check(status)
+    a_arr = np.sort(a_arr)
+    zs = 1.0 / a_arr - 1
+    zs = np.clip(zs, 0, np.inf)
+
+    # deal with normalization
+    if np.isfinite(cosmo["A_s"]):
+        A_s_fid = cosmo["A_s"]
+    elif np.isfinite(cosmo["sigma8"]):
+        # in this case, CCL will internally normalize for us when we init
+        # the linear power spectrum - so we just get close
+        A_s_fid = 2.43e-9 * (cosmo["sigma8"] / 0.87659)**2
+    else:
+        raise CCLError(
+            "Could not normalize the linear power spectrum! "
+            "A_s = %f, sigma8 = %f" % (
+                cosmo['A_s'], cosmo['sigma8']))
+
+    # init isitgr params
+    cp = isitgr.model.CAMBparams()
+
+    # turn some stuff off
+    cp.WantCls = False
+    cp.DoLensing = False
+    cp.Want_CMB = False
+    cp.Want_CMB_lensing = False
+    cp.Want_cl_2D_array = False
+    cp.WantTransfer = True
+
+    # basic background stuff
+    h2 = cosmo['h']**2
+    cp.H0 = cosmo['h'] * 100
+    cp.ombh2 = cosmo['Omega_b'] * h2
+    cp.omch2 = cosmo['Omega_c'] * h2
+    cp.omk = cosmo['Omega_k']
+#    not needed for now
+#    cp.parameterization = 'muSigma'
```

Comment:
These comments are not needed, just the one which points out that GR=1 means MG, while GR=0 means GR

Reply:
Done.
Thanks again for the comments.